core/src/test/resources/HistoryServerExpectations/stage_with_speculation_summary_expectation.json

```diff
@@ -0,0 +1,7110 @@
+[ {
```

Comment:
Why does this file need to be +7000 lines?
Currently, `stage_task_list_w__offset___length_expectation.json` has the maximum number of lines among files under `HistoryServerExpectations` and it's 2451 lines.
Can you minimize this file?